### DISCUSSION TITLE
Allow Images to be fetched from a custom directory

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#779] [Feature Requests]
A user doesn't wish to keep images inside a git repo, they wish to be able to sync them using a custom method. They would like a manner to set the images base path, so that markdown rendering of images will continue to work via GitJournal.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by mnowotnik** [comment]
Also,
A user should be able to choose any media file from device. If the file is not at the images base path, it should be copied there. This would reduce friction of creating notes with images. (Relevant idea: detection of unused media files at base path).

**Comment by vHanda** [comment]
> Also,
> A user should be able to choose any media file from device. If the file is not at the images base path, it should be copied there. This would reduce friction of creating notes with images. (Relevant idea: detection of unused media files at base path).

I'm a bit confused about how this is different than the current behaviour. Right now when you choose any image from the Gallery, that image is copied.

**Comment by mnowotnik** [comment]
> > Also,
> > A user should be able to choose any media file from device. If the file is not at the images base path, it should be copied there. This would reduce friction of creating notes with images. (Relevant idea: detection of unused media files at base path).
> 
> I'm a bit confused about how this is different than the current behaviour. Right now when you choose any image from the Gallery, that image is copied.

Ah, disregard my comment then.
